src/pages/index.js

```diff
@@ -21,91 +34,155 @@ popUpImageModal.setEventListeners();
 function handleCardClick(name, link) {
   popUpImageModal.openModal(name, link);
 }
-
+//Popup for deleting cards from API along with handler
+const dltPopup = new PopupWithForm(constants.selectors.deleteSelector);
+dltPopup.setEventListeners();
+function handleDelete(id, card) {
+  dltPopup.openModal();
+  dltPopup.setSubmitAction(() => {
+    api
+      .deleteCards(id)
+      .then(() => {
+        dltPopup.closeModal();
+        card.deleteUI();
+      })
+      .finally(() => {
+        constants.dltFormSubmit.innerText = "Yes";
+      });
+  });
+}
+//Handler for the Like button
+function handleToggleLikes(id, isLiked, card) {
+  if (isLiked === false) {
+    card.handleLike();
+    isLiked = true;
+    api.toLike(id);
+  } else {
+    card.handleLike();
+    isLiked = false;
+    api.notLiked(id);
+  }
+}
 //FormValidation
-<<<<<<< Updated upstream
+//profile name/description
 const editFormValidator = new FormValidator(
-=======
-const editFormValidator = new FormValidator(settings, profileEditForm);
-editFormValidator.enableValidation();
-const addFormValidator = new FormValidator(settings, addModalForm);
-addFormValidator.enableValidation();
-
-const selectors = {
-  cardSection: ".gallery__cards",
-  cardTemplate: "#card-template",
-  popUpSelector: ".modal",
-  profileSelector: "#profile-edit-modal",
-  addSelector: "#add-modal",
-  imageSelector: ".gallery__card-image",
-  nameSelector: ".profile__name",
-  jobSelector: ".profile__description",
-  avatarSelector: ".profile__avatar",
-};
-const avatarFormValidator = new FormValidator(
->>>>>>> Stashed changes
   constants.settings,
   constants.profileEditForm
 );
-<<<<<<< Updated upstream
 editFormValidator.enableValidation();
+//avatar Form - link
+const avatarFormValidator = new FormValidator(
+  constants.settings,
+  constants.avatarForm
+);
+avatarFormValidator.enableValidation();
+//card form place/Url
 const addFormValidator = new FormValidator(
   constants.settings,
   constants.addModalForm
 );
 addFormValidator.enableValidation();
-
-=======
-avatarFormValidator.enableValidation();
->>>>>>> Stashed changes
 // call queryselector on input inside form /set values
 function handleFormFill(userInfoList) {
```

Comment:
There are 3 popups in the project with buttons and 2 forms. You need to **name** the elements the way all could understand what they are. And the functions as well


`fillProfileInputs`